index.php

```diff
@@ -13,10 +13,23 @@
 		sleep(2);
 		if(!checkAuthentication(trim(strval($_POST['user'])), trim(strval($_POST['pass'])))){
 			echo "<span class='error'>" . AUTHENTICATION_ERROR_CREDENTIAL . "</span>";
+			#sending a log to the syslog when authentication is incorrect
+			$sock = socket_create(AF_INET, SOCK_DGRAM, SOL_UDP);
+			$msg = "kodi_authd Client authentication failure: {$_SERVER['HTTP_X_FORWARDED_FOR']} ({$_POST['user']})";
```

Comment:
The `$_SERVER['HTTP_X_FORWARDED_FOR']` and `$_POST['user']` are defined client(attacker)-side, so it's necessary to add more control / check / sanitization over these variables to prevent **log injection attack**.